server/schedule/operator/builder_test.go

```diff
@@ -496,10 +495,40 @@ func (s *testBuilderSuite) TestBuild(c *C) {
 				AddLearner{ToStore: 3},
 			},
 		},
+		{
+			// issue: https://github.com/tikv/pd/issues/4411
+			// "(enable JointConsensus) remove 1 voter from 2 voter replicas raft group
+			true,
+			[]*metapb.Peer{{Id: 1, StoreId: 1}, {Id: 2, StoreId: 2}},
+			[]*metapb.Peer{{Id: 1, StoreId: 1}},
+			OpRegion,
+			[]OpStep{
+				ChangePeerV2Enter{
+					PromoteLearners: []PromoteLearner{},
+					DemoteVoters:    []DemoteVoter{{ToStore: 2}},
+				},
+				RemovePeer{FromStore: 2},
+			},
+		},
+		{
+			// issue: https://github.com/tikv/pd/issues/4411
+			// "(enable JointConsensus) remove 1 voter from 2 voter replicas raft group
+			true,
+			[]*metapb.Peer{{Id: 1, StoreId: 1}, {Id: 2, StoreId: 2}},
+			[]*metapb.Peer{{Id: 2, StoreId: 2}},
+			OpLeader | OpRegion,
+			[]OpStep{
+				TransferLeader{FromStore: 1, ToStore: 2},
+				ChangePeerV2Enter{
+					PromoteLearners: []PromoteLearner{},
+					DemoteVoters:    []DemoteVoter{{ToStore: 1}},
+				},
+				RemovePeer{FromStore: 1},
+			},
+		},
 	}
 
 	for _, tc := range cases {
-		c.Log(tc.name)
```

Comment:
Why remove it?

Reply:
It prints unnecessary and malformed logs.

Reply:
I made a mistake, I will roll back this part of the edit